From 702260c5a256a742d58cac5d5182d1821c21a79d Mon Sep 17 00:00:00 2001
From: linuxlion <101433270+linuxlion@users.noreply.github.com>
Date: Tue, 21 Nov 2023 18:38:54 -0500
Subject: [PATCH] Update 60-autosuspend.hwdb (#30131)

* Update 60-autosuspend.hwdb

Framework provides expansion cards. For the HDMI and DisplayPort, these benefit power management via enabling auto suspend.

(cherry picked from commit 9023630cb7025650aa4d01ee794b0bb68bfdf2c1)

Related: RHEL-5950
---
 hwdb.d/60-autosuspend.hwdb | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/hwdb.d/60-autosuspend.hwdb b/hwdb.d/60-autosuspend.hwdb
index cfb6084671..1bdc041bb9 100644
--- a/hwdb.d/60-autosuspend.hwdb
+++ b/hwdb.d/60-autosuspend.hwdb
@@ -46,6 +46,16 @@
 usb:v058Fp9540*
  ID_AUTOSUSPEND=1
 
+#########################################
+# Framework
+#########################################
+
+# HDMI Expansion Card
+usb:v32ACp0002*
+# DisplayPort Expansion Card
+usb:v32ACp0003*
+ ID_AUTOSUSPEND=1
+
 #########################################
 # Lenovo
 #########################################
